routing-practice/src/App.js

```diff
@@ -0,0 +1,34 @@
+import logo from './logo.svg'
+import './App.css'
+import ReactDOM from "react-dom/client";
```

Comment:
import sequence should be like 
1 third party modules
2 custom component
3 helper files
4 constant files
5 css, image, icons files 
there should be blank space in between every sections